API/Controllers/CustomerController.cs

```diff
@@ -9,50 +9,49 @@ namespace API.Controllers
     [ApiController]
     public class CustomerController : ControllerBase
     {
-        private readonly ICustomerService _customerService;
+        private readonly ICustomerAppService _appService;
 
-        public CustomerController(ICustomerService customerService)
+        public CustomerController(ICustomerAppService appService)
         {
-            _customerService = customerService ?? throw new ArgumentNullException(nameof(customerService));
+            _appService = appService ?? throw new ArgumentNullException(nameof(appService));
         }
 
         [HttpGet]
-        public IActionResult Get()
+        public virtual IActionResult Get()
         {
```

Comment:
Utilizaremos essa controller somente aqui, e ela não será sobrescrita, por isso, o uso do virtual não se faz necessário.
```suggestion
        public IActionResult Get()
        {
```